AutoRaise.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,348 @@
+// !$*UTF8*$!
+{
+	archiveVersion = 1;
+	classes = {
+	};
+	objectVersion = 48;
+	objects = {
+
+/* Begin PBXBuildFile section */
+		C501B69525875F32005B3964 /* LICENSE.md in Resources */ = {isa = PBXBuildFile; fileRef = C501B69425875F32005B3964 /* LICENSE.md */; };
+		C557DD431B988ED300E59BC9 /* AppDelegate.swift in Sources */ = {isa = PBXBuildFile; fileRef = C557DD421B988ED300E59BC9 /* AppDelegate.swift */; };
+		C557DD451B988ED300E59BC9 /* Images.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = C557DD441B988ED300E59BC9 /* Images.xcassets */; };
+		C557DD481B988ED300E59BC9 /* MainMenu.xib in Resources */ = {isa = PBXBuildFile; fileRef = C557DD461B988ED300E59BC9 /* MainMenu.xib */; };
+		C59A84F02586387800AD9D56 /* AutoRaise in Resources */ = {isa = PBXBuildFile; fileRef = C59A84EF2586387800AD9D56 /* AutoRaise */; };
+/* End PBXBuildFile section */
+
+/* Begin PBXFileReference section */
```

Comment:
no need to add the README.md? I suggest you add a special section to the README on how to start AutoRaise with your launcher and how to compile it from the command line.